reference/uitpas.json

```diff
@@ -3624,9 +3619,14 @@
                       "type": "string",
                       "description": "If redeemable if false, this field contains the reason why.",
                       "enum": [
-                        "DATE",
-                        "VOLUME",
-                        "PASSHOLDER"
+                        "INVALID_CARD",
+                        "PASSHOLDER_VOLUME",
+                        "PASSHOLDER_POINTS",
+                        "PASSHOLDER",
+                        "REWARD_PERIOD",
+                        "REWARD_VOLUME",
+                        "PASSHOLDER_APPLICABLE_CARDSYSTEMS",
+                        "SOCIAL_TARIFF_EXPIRED"
```

Comment:
Some of these are not clear to me what they mean exactly. For example `PASSHOLDER`. And others like `REWARD_VOLUME` I personally can guess but might not 100% clear either for integrators. Is this enum important? Could we add some details in the `description`? Or maybe make the names a bit clearer?

Reply:
the description will contain a user readable message, but I guess it makes sense to add an explanation for all `reason`s as well

Reply:
Looks good aside from two typo's (added a suggestion to fix them)